git-ssh.md

```diff
@@ -0,0 +1,75 @@
+# Installing SSH Key on Macbook for GitHub access
+
+I'm not going to walk through the reasoning behind using SSH keys as authorization or why GitHub is making this change. Let's just dive into the process and get the SSH keys created, installed, and referenced on GitHub.
+
+## Create the SSH Key
+1. Open terminal. **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**
+1. Paste the following code into the terminal, substituting in your GitHub account email address for the `[your GitHub email]`:
+  ```
+  $ ssh-keygen -t -rsa -b 4096 -C "[your GitHub email]"
+  ```
+3. This generates a new SSH key, using the provided email as a label.
+  ```
+  > Generating public/private [your GitHub email] key pair.
+  ```
+4. When your promted to "Enter a file in which to save the key," press `Enter`. This accepts the default location.
+```
+> Enter a file in which to save the key (/Users/you/.ssh/id_youremailaddress):
+```
+5. At the prompt, type a secure passphrase. (This is optional. Using a passphrase will require you to enter the passphrase each time you attempt to authenticate with GitHub.)
```

Comment:
I would say that we explicitly tell them _not_ to set a passphrase. It's just one more thing we'll have to troubleshoot with them.

So I might rephrase steps 4/5 like:

----

When prompted, press the _return_ key to accept the default values:

```
Enter file in which to save the key (/Users/someuser/.ssh/id_ed25519): [Press return]
Enter passphrase (empty for no passphrase): [Press return]
Enter same passphrase again: [Press return]
```